File: 20190420_TPBC2019/B-eeeee.py
def main():
    N = int(input())
    S = input()
    K = int(input())
    unchange = S[K-1]
    for i in range(N):
        if S[i] == unchange:
            pass
        else:
            S = S.replace(S[i], '*')
    print(S)


if __name__ == '__main__':
    main()

File: 20190420_TPBC2019/B-eeeee.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    int N, K;
    string S;
    cin >> N;
    cin >> S;
    cin >> K;
    for (int i = 0; i < N; i++) {
        if (S[i] == S[K-1]) {}
        else {
            S.replace(i, i+1, "*");
        }
    }
    cout << S << endl;
    return 0;
}
